coral/src/app/features/topics/request/TopicRequest.test.tsx

```diff
@@ -410,4 +439,43 @@ describe("<TopicRequest />", () => {
       });
     });
   });
+
+  describe("AdvancedConfiguration", () => {
```

Comment:
Small nitpick, but I would also test here that the headline, the explanation text, and the correct link is rendered. It's a tiny detail, but I think since it's important for users to get that information, it's worth confirming it in a test.